var/spack/repos/builtin/packages/xxdiff/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Xxdiff(MakefilePackage):
+    """Graphical File And Directories Comparator And Merge Tool."""
+
+    homepage = "https://furius.ca/xxdiff/"
+    git = "https://github.com/blais/xxdiff.git"
+
+    maintainers("vanderwb")
+
+    version("master", branch="master")
+    version("2023-01-10", commit="604300e")
```

Comment:
We should store the full commit SHA here since it's possible for [short commit hashes to collide](https://stackoverflow.com/a/56012939).

Reply:
Thanks @alecbs. Change made. Note that the Spack doc linked by Tammy seems to say this is okay, though perhaps I misinterpreted something:

> This doesn’t have to be a full hash; you can abbreviate it as you’d expect with git:
>
>version("2014-10-08", commit="9d38cd")